packages/dds/sequence/src/intervals/sequenceInterval.ts

```diff
@@ -38,13 +38,13 @@ import {
 } from "../intervalCollection.js";
 
 import {
-	IIntervalHelpers,
 	ISerializableInterval,
 	ISerializedInterval,
 	IntervalStickiness,
 	IntervalType,
 	endReferenceSlidingPreference,
 	startReferenceSlidingPreference,
+	type IIntervalHelpers,
 } from "./intervalUtils.js";
```

Comment:
I thought the sorting would still want `type IIntervalHelpers` first.
Other pattern that has no such confusion is to have separate `import type {}` block. We do that more now.

Reply:
i don't know. i just let the formatter do what it wants.